### PR TITLE
Datawave helm chart integration

### PR DESCRIPTION
**Pull request by FineAndDandy**
Add integration and interoperability to the datawave-helm-charts repo, support added in https://github.com/NationalSecurityAgency/datawave-helm-charts/pull/64

This updates the included example values.yaml in the umbrella chart to be easily compatible with datawave-helm-charts

### CODE REVIEW

File: bundler/src/main/java/datawave/microservice/bundler/BundleFileScanner.java [hunk 1]
```diff
@@ -1,9 +1,12 @@
 package datawave.microservice.bundler;
 
-import datawave.ingest.mapreduce.job.BulkIngestKey;
-import datawave.microservice.bundler.configuration.BundlerProperties;
-import datawave.microservice.file.FileScanner;
-import datawave.microservice.file.configuration.FileScannerProperties;
+import java.io.*;
```
Comment (alerman) [inline bundler/src/main/java/datawave/microservice/bundler/BundleFileScanner.java:3]: Star imports. You should be ashamed.

**Review by alerman — APPROVED** [review]
LGTM. I didnt run myself so I assume you did :smile:

**Comment by FineAndDandy** [comment]
I can confirm I ran this with https://github.com/NationalSecurityAgency/datawave-helm-charts/pull/64 and then successfully ingested using the myjson feeder

File: chart/README.MD
Comment (FineAndDandy) [inline chart/README.MD:1]: These steps are required when running with the datawave-helm-charts in order to deploy and reference the images locally

**Review by cfkoehler — APPROVED** [review]
Looks good. Got it all working.

**Review by alerman — APPROVED** [review]

**Pull request merged**